src/liquid/components/ld-sidenav/ld-sidenav-navitem/ld-sidenav-navitem.tsx

```diff
@@ -50,7 +50,7 @@ export class LdSidenavNavitem implements InnerFocusable {
   @Prop() mode?: 'secondary' | 'tertiary'
 
   /** Applies full border-radius. */
-  @Prop() rounded = false
+  @Prop({ reflect: true }) rounded = false
```

Comment:
What's the advantage of reflecting prop changes here?

Reply:
I'm using the attribute as a selector in ld-sidenav-accordion.

Reply:
I'm using the attribute as a selector in ld-sidenav-accordion.